lib/global-styles.php

```diff
@@ -119,11 +120,12 @@ function gutenberg_experimental_global_styles_settings( $settings ) {
 	// In the site editor, the user can change styles, so the client
 	// needs the ability to create them. Hence, we pass it some data
 	// for this: base styles (core+theme) and the ID of the user CPT.
-	$screen = get_current_screen();
+	$screen = ! $is_rest_request ? get_current_screen() : null;
```

Comment:
These changes are pending to be updated after this https://github.com/WordPress/gutenberg/issues/30082#issuecomment-816679803